config/domains.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Foundry convention is converging on putting this in `script/input/{chainId}/{name}.json` so `script/input/5/config.json` in this case.

Reply:
The chain id is the root chain (Mainnet or Goerli). I'm following some conventions from other Foundry repos as these standards get built.

Maybe since this repo is entirely Goerli an exception can be made, but I don't see any issue with keeping the convention either.

Reply:
https://github.com/makerdao/spells-goerli/pull/132/commits/e3a857bce192819982700b9e3b402a358fa08fb9